lib/eex/lib/eex.ex

```diff
@@ -76,6 +76,12 @@ defmodule EEx do
         This will never appear
       <% end %>
 
+  To escape eex in eex use `<%% content %>`. For example:
+
+      <%%= x + 3 %>
+
+  Will be rendered as `<%= x + 3 %>`
```

Comment:
```suggestion
  will be rendered as `<%= x + 3 %>`
```

Reply:
I will avoid the used of code blocks for code that would fit in a single line.

Reply:
I like the separate code block because I think it will enhance the scannability of the docs. In same fashion I would consider the "Will" on this line to be starting a new sentence (and hence keep the capitalization as-is). I am willing to be overruled but I wanted to state/defend my opinion.

Reply:
I agree code blocks are better here.